package.json

```diff
@@ -19,8 +19,8 @@
     "dataframe": "^1.3.0",
     "envify": "^3.2.0",
     "lodash": "^4.1.0",
-    "react": ">=0.12.2",
-    "react-dom": "^0.14.7",
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1",
```

Comment:
No, this is just a product of `npm install react@latest`. `create-react-class` requires a bump in react version.